### DISCUSSION TITLE
CF代理后还是慢的解决办法

### DISCUSSION THREAD

**Discussion by yujielab** [AlistGo/alist#6481] [Show and tell]
fix 这是一个修复
在cloudflare页面→(轻触你的域名)进入域名页→添加dns记录(A)你想要的前缀如www.baidu.com(www就是前缀)，你可以设置想要的前缀→ip填(8.8.8.8)→开小云朵代理☁️。

tips:在域名页(work)加路由路线，填你23步设置的完整域名，如你的前缀为star，域名为baidu.com，你的完整域名是star.baidu.com，填(来源)为你设置的加速work名字。可以参考怎么改cf优选ip

灵感:因为被中国制裁了→试了cf优选ip法(临时有效)一会就不行了→又调了一下(发现要么联通可以移动不信要么其他什么什么的)→又试了cf st测试出优选ip(还是不可以，除了测试的那台机器正常一点，速度还是≤2mb/s)→想说直接指向dns服务器144.144.144.144加上cf的代理(完美)跑到了≤9mb/s→改8.8.8.8后飞起满速了

简单来说就是和cf优选ip法一样，只不过不需要选ip，直接填8.8.8.8就可以了，记得开云☁️朵，原理不知道但是好用！

**Comment by ypq123456789** [comment]
开云朵不管你填什么ip都是cf自动给你分配cdn的ip，可能暂时很快，但过段时间就不快了，和优选还是不一样的。
